osc-server/src/main/java/org/osc/core/broker/model/plugin/PluginTracker.java

```diff
@@ -108,10 +131,32 @@ public void installableUnitsChanged(Collection<InstallableUnitEvent> events) {
 
         };
 
-        this.serviceTracker = new ServiceTracker<T,T>(context, pluginClass, null) {
+        this.serviceTrackerManager = new ServiceTracker<T,T>(context, pluginClassManager, null) {
             @Override
             public T addingService(ServiceReference<T> reference) {
-                if (!containsRequiredProperties(reference)) {
+                if (!containsRequiredProperties(reference, REQUIRED_MANAGER_PLUGIN_PROPERTIES)) {
```

Comment:
can you avoid the replicated code and make it common and just call it here and below.